src/fragments/lib/troubleshooting/common/upgrading.mdx

```diff
@@ -46,88 +46,77 @@ In order to use Amplify JavaScript V6, you will need to make sure you are using
 
 ## Step 2: Upgrade your Amplify project dependencies
 
-<Callout>
-  Make sure to delete your package-lock.json and node_modules folder before
-  running `npm install`
-</Callout>
+<Callout>Make sure to delete your package-lock.json and node_modules folder before running `npm install`</Callout>
 
 <BlockSwitcher>
+  <Block name="Web">
+    Upgrade to the latest Amplify library using the following command.
 
-<Block name="Web">
-
-Upgrade to the latest Amplify library using the following command.
-
-```
-npm install aws-amplify
-```
-
-The `aws-amplify` package in v6 includes the categories below:
-
-- Auth
-- API
-- Storage
-- Analytics
-- DataStore
-- In-App Messaging
-
-If you previously listed `@aws-amplify` namespaced packages for the above categories as dependencies in your `package.json`, you will need to remove them to avoid dependency duplication.
+    ```
+    npm install aws-amplify
+    ```
 
-If you would like to use Geo, Predictions, PubSub, or Interactions you will need to install those packages separately. (see [category-specific migration instructions](/[platform]/build-a-backend/troubleshooting/migrate-from-javascript-v5-to-v6/#4-upgrade-category-usage))
+    If you are using npm as a package manager, make sure the version of `aws-amplify` in your package.json file is `latest` or `^6.0.1` and re-installing dependencies by running `npm install`.
```

Comment:
IIRC `6.0.1` had an issue and `6.0.2` was used as `latest` for the MV release (@jimblanc probably remembers better than me):
```suggestion
    If you are using npm as a package manager, make sure the version of `aws-amplify` in your package.json file is `latest` or `^6.0.2` and re-installing dependencies by running `npm install`.
```

Reply:
Good catch, that's correct `6.0.2` was what we flipped `latest` to at launch.

Reply:
@erinleigh90 the version here still needs to be updated 😅